src/GridRegistry.ts

```diff
@@ -0,0 +1,47 @@
+import { WidgetBaseConstructor } from '@dojo/widget-core/interfaces';
+import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
+import Body, { BodyInterface } from './Body';
+import Cell, { CellInterface } from './Cell';
+import Footer, { FooterInterface } from './Footer';
+import Header, { HeaderInterface } from './Header';
+import HeaderCell, { HeaderCellInterface } from './HeaderCell';
+import Row, { RowInterface } from './Row';
+
+export interface GridRegistered {
+	[key: string]: WidgetBaseConstructor;
+	body: BodyInterface;
```

Comment:
Seeing it implemented, yes, these should either be `WidgetBaseConstructor<BodyProperties>` or `typeof Body`... I would prefer the latter as it would be the full interface.